lib/autoload/course/assessment/programming_test_case_report.rb

```diff
@@ -225,7 +225,11 @@ def messages
     # @return [String]
     def get_meta_attribute(attribute_name)
       meta = @test_case.search('meta')
-      meta.present? && meta[0][attribute_name] ? meta[0][attribute_name] : ''
+      if meta.present? && meta[0][attribute_name]
```

Comment:
Ok, let's also rename the function to `get_testcase_metadata`?

It's supposed to be JUnit XML, but I'm not sure how consistent implementations are.